pkg/apis/kops/cluster.go

```diff
@@ -136,6 +136,12 @@ type ClusterSpec struct {
 	//   missing: default policy (currently OS security upgrades that do not require a reboot)
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 
+	// Install Helm
+	// Valid values:
+	//   'true' Install the tiller manifest to enable helm https://github.com/kubernetes/helm
+	//   'false' Don't install it
+	EnableHelm *bool `json:"enableHelm,omitempty"`
```

Comment:
I'm kind of meh on this being part of the API surface. Should we instead accept a list of add-ons that we launch, possibly with "symbolic" add-ons to e.g. launch a stable version of helm, dashboard, etc?